conda/recipes/dask-cudf/meta.yaml

```diff
@@ -23,12 +23,12 @@ requirements:
   host:
     - python
     - cudf {{ version }}
-    - dask>=2021.3.1
+    - dask==2021.3.1
     - distributed >=2.22.0
```

Comment:
Need to constrain this to `>=2.22.0,<=2021.3.1` across conda recipes / environments